README.md

```diff
@@ -7,7 +7,7 @@ This project is currently under development. Use at your own risk. :)
 
 <p align="center">
   <a href="https://github.com/aidotse/PASEOS/">
-    <img src="resources/images/datastructure.svg" alt="Scheme"  width="910" height="459">
+    <img src="paseos/resources/images/datastructure.svg" alt="Scheme"  width="910" height="459">
```

Comment:
Let's keep the readme images (this and below 2) in the top-level folder resources, there is no need to ship them with the module and otherwise they have no reason to be in the module folder